Writerside/topics/ChiselMethod.md

```diff
@@ -1,3 +1,24 @@
 # The Chisel Method
 
-_To be written_
\ No newline at end of file
+The Chisel method for writing parsers allows to build parsers consistently using the same approach which allows to achieve a consistency on the architectural decisions and enables developers to get familiar with a project easily.
+
+This method is called Chisel because building parsers is about getting the information out of the code, as when you use a chisel you "take" the statue out of the marble. Also, since Strumenta means tools in Latin, Chisel takes the place as one of the most important tool in the company's toolset.
+
+The Chisel method is based on the following principles:
+1. We define a clear goal, which is shared by the Client and the Language Engineering Team. This goal is objective, and it is not subject to interpretation.
```

Comment:
For lists to work in markdown I am not sure if we need an empty line before the first point